services/harmony/test/ignore-errors.ts

```diff
@@ -560,7 +560,7 @@ describe('ignoreErrors', function () {
       });
     });
 
-    describe('When a request spans multiple CMR pages', function () {
+    describe('CDD When a request spans multiple CMR pages', function () {
```

Comment:
remove `CDD `

Reply:
I knew there would be at least one of these - thanks